### ISSUE TITLE
error: no such bucket on a new fresh project

### ISSUE THREAD

**Issue by veritem** [sst/sst#4446]
Hey, thank you so much for this library!!!

I have played around with ion 2 months ago, which really was nice. however, I'm trying it again and getting an error that I don't know what's going on. 

<img width="1067" alt="image" src="https://github.com/sst/ion/assets/53856673/fdcc6b60-189d-4939-a9f0-8a54ae15c6ef">

this error is showing up on every sst ion project I create on my laptop. I guess it has to do with my configurations, but I can't tell which. I have all my aws  credentials in the right place. Has anyone ever dealt with something like this? PS: I'm not even using s3 bucket in my projects.

**Comment by NeatFastro** [comment]
I was getting this error:
`The state bucket is missing, it may have been accidentally deleted. Go to https://console.aws.amazon.com/systems-manager/parameters/%252Fsst%252Fbootstrap/des
cription?tab=Table and check if the state bucket mentioned there exists. If it doesn't you can recreate it or delete the `/sst/bootstrap` key to force recreation.`

So as described I deleted the the `/sst/bootstrap` and the rerun the `npx sst dev` command and it worked this time!

**Comment by JohnLozier** [comment]
I was in the process of moving from sst on wsl to native windows and am getting the same error. I checked the parameter and the buckets referenced exist and are in the same region. I tried deleting `/sst/bootstrap` and rerunning, but the error still persists despite the `/sst/bootstrap` key not being recreated. 
```
The state bucket is missing, it may have been accidentally deleted. Go to https://console.aws.amazon.com/systems-manager/parameters/%252Fsst%252Fbootstrap/description?tab=Table and check if the state bucket mentioned there exists. If it doesn't you can recreate it or delete the `/sst/bootstrap` key to force recreation.
```

This is only happening when running on Windows. The exact same code works fine when run on wsl. I also noticed that windows is defaulting to a different stage name, but the issue still occurs when passing in the same stage wsl is using.
